download/download.py

```diff
@@ -181,19 +181,33 @@ def _fetch_file(url, file_name, resume=True,
                         ff.write(chunk)
         else:
             # Check file size and displaying it alongside the download url
-            u = urllib.request.urlopen(url, timeout=timeout)
+            req = urllib.request.Request(
```

Comment:
Since these are all the same, could we just make this a little function like:

```
def request_agent(url):
    req = urllib.request.Request(
                url,
                data = None, 
                headers = {
                    'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.47 Safari/537.36'
                }
            )
    return req
```

and then do

```
req = request_agent(url)
```
and proceed as normal?

Reply:
done